FLEDGE.md

```diff
@@ -240,9 +244,38 @@ The `ads` list contains the various ads that the interest group might show.  Eac
 
  * `adRenderId`: A short [DOMString](https://webidl.spec.whatwg.org/#idl-DOMString) up to 12 characters long serving as an identifier for this ad in this interest group. When this field is specified it will be sent instead of the full ad object for [B&A server auctions](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md).
 
- * `buyerAndSellerReportingId`: If set, the value is used instead of the interest group name or `buyerReportingId` for reporting in `reportWin` and `reportResult`. Note that this field needs to be jointly k-anonymous with the interest group owner, bidding script URL, and render URL to be provided to these reporting fuctions (in the same way that the interest group name would have needed to be).
+ * `selectableBuyerAndSellerReportingIds`: An array of strings, one of which may be
+   selected by `generateBid()` to be reported to `reportWin()` and `reportResult()`
+   along with `buyerAndSellerReportingId` and `buyerReportingId` when all three values
+   are jointly k-anonymous along with the interest group owner, bidding script URL,
+   and render URL to be provided to these reporting functions. See
+   [Reporting IDs](#54-reporting-ids-in-reporting) for more details.
+
+ * `buyerAndSellerReportingId`: If set and `selectableBuyerAndSellerReportingIds` is
+   unset, the value is used instead of the interest group name or `buyerReportingId`
```

Comment:
@orrb1, do we need to change this from "is unset" to "is unset or not selected"?

Reply:
Yes, that's more precise, here and below on 267. Thanks.

Reply:
Oops, the later comment is a bit lower down, so unresolving for now.